### PR TITLE
Added Tutorial for RedisInsight on Windows 

### PR DESCRIPTION
**Pull request by ajeetraina**
Added tutorial for Windows + RedisInsight

### CODE REVIEW

File: docs/explore/redisinsightv2/windows/index-windows.mdx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+sidebar_label: How to run RedisInsight on Windows
+authors: [ajeet]
+---
+
+RedisInsight is a visual tool that provides capabilities to design, develop and optimize your Redis application. It is a 100% free Redis GUI that allows developers like you to interact with your databases and manage your data.
+
+![featured image](images/redisinsight_photo.png)
+
+
+RedisInsight v2.0 incorporates a completely new tech stack based on the popular Electron and Elastic UI frameworks. You can run the application locally along with your favorite IDE, and it remains cross-platform, supported on Linux, Windows, and MacOS.RedisInsight Browser lets you explore keys in your Redis server. You can add, edit and delete a key. You can even update the key expiry and copy the key name to be used in different parts of the application.
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/windows/index-windows.mdx]:
```suggestion
RedisInsight v2.0 incorporates a completely new tech stack based on the popular Electron and Elastic UI frameworks. You can run the application locally along with your favorite IDE, and it remains cross-platform, supported on Linux, Windows, and MacOS. RedisInsight Browser lets you explore keys in your Redis server. You can add, edit and delete a key. You can even update the key expiry and copy the key name to be used in different parts of the application.
```

File: docs/explore/redisinsightv2/windows/index-windows.mdx [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+## RedisInsight Windows Installer
+
+The RedisInsight desktop client installer for Windows is just 70 MB in size. It allows you to download and use the RedisInsight GUI locally. The desktop client is supported on Windows operating systems and works with all variants of Redis. RedisInsight should install and run on a fresh Windows system.
+
+:::info INFO
+There is no need to install any .NET framework in order to install RedisInsight on Windows platform.
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/windows/index-windows.mdx]:
```suggestion
There is no need to install the .NET framework in order to install RedisInsight on Windows.
```

File: docs/explore/redisinsightv2/windows/index-windows.mdx [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+[Click this link](https://redis.com/redis-enterprise/redis-insight/#insight-form) to access a form that allows you to select the operating system of your choice(in our case, it's Microsoft Windows).
+
+
+![form to fill to download RedisInsight](images/redisinsight-form.png)
+
+Fill out the rest of the form and click “Download”.  
+
+#### Install RedisInsight
+
+Click on the RedisInsight executable(in the form of .exe) and install it in your system.
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/windows/index-windows.mdx]:
```suggestion
Click on the RedisInsight executable (.exe file) and install it in your system.
```

File: docs/explore/redisinsightv2/windows/index-windows.mdx [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+![setup redisinsight](images/redisinsight-setup.png)
+
+Once the RedisInsight software is installed, click on its icon to open the RedisInsight application. It will display the End-User License Agreement and Privacy Settings. Enable Analytics and Encrypt sensitive information as per your preference.
+
+![accept redisinsigh licence](images/redisinsight-eula.png)
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/windows/index-windows.mdx]:
```suggestion
![accept redisinsight licence](images/redisinsight-eula.png)
```

File: docs/explore/redisinsightv2/windows/index-windows.mdx [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+
+### Step 3. Connect to a Redis Database
+
+Enter the requested details, including Host (endpoint), Port, and Alias in the form, as shown below. You can use "default" as username for now. Then click “ADD REDIS DATABASE”.
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/windows/index-windows.mdx]:
```suggestion
Enter the requested details, including Host (endpoint), Port, and Alias in the form, as shown below. You can use "default" as the username for now. Then click “ADD REDIS DATABASE”.
```

File: docs/explore/redisinsightv2/windows/index-windows.mdx [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+*   `country` : Country Name.
+*   `country_code` : Country Code.
+*   `city` : City of the user.
+*   `longitude` : Longitude of the user.
+*   `latitude` : Latitude of the user.
+*   `last_login` : Epoch time of the last login.
+
+### Step 6. Clone the repository
+
+Open up the CLI terminal and run the following command:
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/windows/index-windows.mdx]:
```suggestion
Open up the CLI terminal and run the following commands:
```

File: docs/explore/redisinsightv2/windows/index-windows.mdx [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+
+ ```bash
+  git clone https://github.com/redis-developer/redis-datasets
+  cd redis-datasets/user-database
+ ```
+
+
+### Step 7. Import the user database keys
+
+Open up the CLI terminal and run the following command:
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/windows/index-windows.mdx]: They'll also need a password to do this for a cloud database, and maybe a reminder to use the host and port of their cloud database instance?

File: docs/explore/redisinsightv2/windows/index-windows.mdx [hunk 8]
```diff
@@ -0,0 +1,173 @@
...
+
+
+![listing the keys](images/redisinsight-keys-added.png "image_tooltip")
+
+
+You can get a real-time view of the data in your Redis database as shown below:
+
+
+
+Select any key in the keys view and the key's value gets displayed in the right hand side that includes Fields and values.
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/windows/index-windows.mdx]:
```suggestion
Select any key in the keys view and the key's value gets displayed in the right hand side that includes fields and values.
```

File: docs/explore/redisinsightv2/windows/index-windows.mdx [hunk 9]
```diff
@@ -0,0 +1,173 @@
...
+
+![hash keys listed](images/redisinsight_hashes.png)
+
+
+### Step 8. Modify a key
+
+RedisInsight browser tool allows you to modify the data instantly.
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/windows/index-windows.mdx]:
```suggestion
The RedisInsight browser tool allows you to modify the data instantly.
```

File: docs/explore/redisinsightv2/windows/index-windows.mdx [hunk 10]
```diff
@@ -0,0 +1,173 @@
...
+Select any key and change the values as shown in the following screenshot
+
+![modify the redis keys](images/redisinsight_modify_keys.png)
+
+
+
+### Step 9. Cleaning up
+
+Run the below command to clean up all the Redis keys
```
Comment (simonprickett) [inline docs/explore/redisinsightv2/windows/index-windows.mdx]:
```suggestion
Run the following command to clean up all the Redis keys:
```

**Review by simonprickett — APPROVED** [review]
Left a small number of minor optional suggestions.

**Comment by ajeetraina** [comment]
@simonprickett This time created a separate branch and now it looks good as I can have separate PR for each of my tutorials. I just added Redis OM links to Create page. Do check it once.

File: docs/explore/redisinsightv2/windows/index-windows.mdx (hunk 7 above)
Comment (ajeetraina) [reply docs/explore/redisinsightv2/windows/index-windows.mdx]: Fixed.

**Pull request merged**